<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.2.2 (Python 3.12.5 on win32)" generated="2025-04-16T12:48:01.717908" rpa="false" schemaversion="5">
<suite id="s1" name="InstallationApplications" source="C:\Users\bahae\Desktop\PfeProject\test_cases\InstallationApplications.robot">
<kw name="DÃ©marrer Driver" type="SETUP">
<kw name="Setup Driver" owner="Integration">
<msg time="2025-04-16T12:48:33.008003" level="FAIL">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
    at getResponseForW3CError (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.js:1143:9)
    at asyncHandler (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.js:485:57)</msg>
<var>${driver}</var>
<arg>${Device}</arg>
<doc>Initialise et retourne le driver Appium.</doc>
<status status="FAIL" start="2025-04-16T12:48:01.957005" elapsed="31.054004">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
    at getResponseForW3CError (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.js:1143:9)
    at asyncHandler (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.js:485:57)</status>
</kw>
<kw name="Set Suite Variable" owner="BuiltIn">
<arg>${driver}</arg>
<doc>Makes a variable available everywhere within the scope of the current suite.</doc>
<status status="NOT RUN" start="2025-04-16T12:48:33.012005" elapsed="0.000000"/>
</kw>
<status status="FAIL" start="2025-04-16T12:48:01.956005" elapsed="31.056000">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
    at getResponseForW3CError (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.js:1143:9)
    at asyncHandler (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.js:485:57)</status>
</kw>
<test id="s1-t1" name="Test Install Apk" line="27">
<status status="FAIL" start="2025-04-16T12:48:33.012005" elapsed="0.000000">Parent suite setup failed:
WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
    at getResponseForW3CError (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.js:1143:9)
    at asyncHandler (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.js:485:57)</status>
</test>
<test id="s1-t2" name="Test Uninstall Application" line="43">
<status status="FAIL" start="2025-04-16T12:48:33.012005" elapsed="0.000000">Parent suite setup failed:
WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
    at getResponseForW3CError (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.js:1143:9)
    at asyncHandler (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.js:485:57)</status>
</test>
<kw name="Fermer Driver" type="TEARDOWN">
<kw name="Close Driver" owner="Integration">
<msg time="2025-04-16T12:48:33.013005" level="FAIL">Variable '${driver}' not found.</msg>
<arg>${driver}</arg>
<doc>Ferme proprement le driver Appium.</doc>
<status status="FAIL" start="2025-04-16T12:48:33.013005" elapsed="0.000000">Variable '${driver}' not found.</status>
</kw>
<status status="FAIL" start="2025-04-16T12:48:33.013005" elapsed="0.000000">Variable '${driver}' not found.</status>
</kw>
<status status="FAIL" start="2025-04-16T12:48:01.718907" elapsed="31.294098">Suite setup failed:
WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms
    at getResponseForW3CError (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\errors.js:1143:9)
    at asyncHandler (C:\Users\bahae\AppData\Roaming\npm\node_modules\appium\node_modules\@appium\base-driver\lib\protocol\protocol.js:485:57)

Also suite teardown failed:
Variable '${driver}' not found.</status>
</suite>
<statistics>
<total>
<stat pass="0" fail="2" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="InstallationApplications" id="s1" pass="0" fail="2" skip="0">InstallationApplications</stat>
</suite>
</statistics>
<errors>
</errors>
</robot>
